projects/soft_999_karim_alatrash_fw_102/src/main.c

```diff
@@ -0,0 +1,48 @@
+//
+// Created by Karim Alatrash on 2021-03-08.
+//
+#include <stdbool.h>     // for booleans
+#include <stdint.h>      // for integer types
+#include "interrupt.h"   // interrupts are required for soft timers
```

Comment:
Technically according to the [coding standards](https://uwmidsun.atlassian.net/wiki/spaces/ELEC/pages/9306218/Coding+Standards) you should have a newline between the <standard> imports and the "midsun-specific" imports. Also I don't think you use `<stdbool.h>` so you can remove it, and you can remove the eclipse autogenerated header comments.